addons/l10n_ar/models/account_journal.py

```diff
@@ -9,14 +9,30 @@ class AccountJournal(models.Model):
     _inherit = "account.journal"
 
     l10n_ar_afip_pos_system = fields.Selection(
-        selection='_get_l10n_ar_afip_pos_types_selection', string='AFIP POS System')
+        selection='_get_l10n_ar_afip_pos_types_selection', string='AFIP POS System',
+        compute='_compute_l10n_ar_afip_pos_system', store=True, readonly=False)
     l10n_ar_afip_pos_number = fields.Integer(
         'AFIP POS Number', help='This is the point of sale number assigned by AFIP in order to generate invoices')
     company_partner = fields.Many2one('res.partner', related='company_id.partner_id')
     l10n_ar_afip_pos_partner_id = fields.Many2one(
         'res.partner', 'AFIP POS Address', help='This is the address used for invoice reports of this POS',
         domain="['|', ('id', '=', company_partner), '&', ('id', 'child_of', company_partner), ('type', '!=', 'contact')]"
     )
+    l10n_ar_share_sequences = fields.Boolean(
```

Comment:
@jjscarafia @zaoral remove this as we don't have it anymore
cc' @fvz-odoo @pablohmontenegro